metamorph/zmq.go

```diff
@@ -166,10 +166,6 @@ func (z *ZMQ) Start() {
 		z.logger.Fatal(err)
 	}
 
-	if err = zmq.Subscribe("invalidtx", ch); err != nil {
```

Comment:
Can we only omit the error `IsMissingInputs`. You can add `continue` in check `if txInfo.IsMissingInputs` so that we still catch error `txInfo.IsDoubleSpendDetected`